docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md

```diff
@@ -21,7 +21,7 @@ Last Updated: 2024-08-07
   "panel": "1",
   "Order": "a040_",
   "Type": "Combobox",
-  "ComboItems": "Default DHCP Google Cloudflare Cloudflare_Malware Cloudflare_Malware_Adult Open_DNS Quad9 AdGuard_Ads_Trackers AdGuard_Ads_Trackers_Malware_Adult",
+  "ComboItems": "Default DHCP Google Cloudflare Cloudflare_Malware Cloudflare_Malware_Adult Open_DNS Quad9 AdGuard_Ads_Trackers AdGuard_Ads_Trackers_Malware_Adult dns0.eu_Open dns0.eu_ZERO dns0.eu_KIDS",
```

Comment:
Do not edit this section of the code. This is auto generated.

Reply:
Done